autodiff/forward/forward.hpp

```diff
@@ -761,13 +772,19 @@ constexpr auto inverse(U&& expr)
 }
 
 //-----------------------------------------------------------------------------
-// AUXILIARY CONSTEXPR CONSTANTS
+// AUXILIARY CONSTEXPR FUNCTIONS TO GET CONSTANTS
 //-----------------------------------------------------------------------------
 template<typename U>
-constexpr auto Zero = static_cast<ValueType<U>>(0);
+constexpr auto Zero()
```

Comment:
Are the introduced changes causing the constexpr template constants to fail somehow? I would say constants are simpler than functions that return a constant.

Reply:
Yes, as I noted in description this constant creation failed since we can’t create posit at compile time. To avoid such behavior for non-literals I introduced function instead constant. Constexpr functions may be evaluated at compile time and in runtime. Hope it is clear =)

Reply:
OK, yes, super clear.